### PR TITLE
Simplify last commit cache

### PR DESCRIPTION
**Pull request by lafriks**

### CODE REVIEW

File: custom/conf/app.ini.sample [hunk 1]
```diff
@@ -648,18 +646,6 @@ ITEM_TTL = 16h
 
 ; Last commit cache
 [cache.last_commit]
-; if the cache enabled
```
Comment (6543) [inline custom/conf/app.ini.sample:651]: @lafriks can wee keep ENABLED for main cache and cache.last_commit section?

Comment (lafriks) [reply custom/conf/app.ini.sample:651]: it duplicates TTL but we can

**Comment by lunny** [comment]
Only a nit. `TTL` is not a good config name for that. And I think 0 should mean it will never expired. So I still think `ENABLED` is better.

File: custom/conf/app.ini.sample (hunk 1 above)
Comment (lafriks) [reply custom/conf/app.ini.sample:651]: @6543 Added back

**Comment by lafriks** [comment]
@lunny added back ;)

**Comment by lunny** [comment]
@lafriks Thanks! And I think the cache ttl default time `16h` is too short. I would like to left the default time for last_time as `8760h` == 1 year.
The last time is a mapping of (commitid, treepath) -> commitid, it's not a branch or tag name. Except one commit was deleted from the repository, there is no reason to expire the cache at the short time.

**Comment by lafriks** [comment]
sure why not, I'm not against it

**Pull request merged**